packages/sdk-router/README.md

```diff
@@ -77,17 +99,28 @@ const bridgeQuotes: BridgeQuote[] = await synapseSDK.allBridgeQuotes(
   tokenOut,
   // Amount of tokens to bridge, in origin token decimals: 1_000_000_000
   amountIn,
-  // Deadline for the transaction to be initiated on the origin chain, in seconds (optional)
-  deadline
+  {
+    // Deadline for the transaction to be initiated on the origin chain, in seconds (optional)
+    deadline: 1234567890,
+    // List of bridge modules to exclude from the result, optional.
+    // Empty list means that all modules are included.
+    excludedModules: ['SynapseBridge', 'SynapseCCTP', 'SynapseRFQ'],
+    // Address of the user on the origin chain, optional.
+    // MANDATORY if a smart contract is going to initiate the bridge operation on behalf of the user.
+    originUserAddress: '0x1234567890abcdef1234567890abcdef12345678',
+  }
 )
 ```
 
-The returned list is sorted by the `maxAmountOut` field, so the first quote is the one yielding the highest amount of tokens on the destination chain.
-
-If either of the input/output tokens is a native gas token (e.g. ETH on Ethereum/Arbitrum, AVAX on Avalanche, etc.), the specialized address `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` should be used instead of the token address.
+- **Important:** It is crucial to provide the `originUserAddress` when a smart contract will initiate the bridge operation on behalf of the user. This requirement applies to smart wallets (e.g., Safe) and third-party integrations (such as bridge aggregator smart contracts).
+- For native gas tokens (e.g., ETH on Ethereum/Arbitrum, AVAX on Avalanche, etc.), use the specialized address `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` instead of the token address for either input or output tokens.
+- The returned list is sorted by the `maxAmountOut` field in descending order, with the first quote offering the highest amount of tokens on the destination chain.
+- All quotes in the list are provided without any slippage applied. To add slippage to the quotes, use the `applyBridgeSlippage` function.
```

Comment:
lets change the language to "without any slippage settings applied" saying "without any slippage" doesnt fully communicate the point imo

Reply:
Fixed in 1813785dedee0b74265614df8cb9ae17cbb8ed7f